h2o-r/tests/testdir_jira/runit_pubdev2801.R

```diff
@@ -0,0 +1,12 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+#Starting H2O server from R ignores IP and port parameters
+#This is a test to see if there is a proper fix in place
+
+test_pubdev2801 <- function(){
+  h2o.init(port = 54355)
```

Comment:
Maybe I'm a little bit detailed but this won't test the actual port, just that it start and close instance. We can add `as.integer(strsplit(h2o.clusterStatus()$h2o$h2o, ":")[[1L]][2L])` and confirm port on which h2o is running.